### PR TITLE
Add links and references for many projects.

### PR DESCRIPTION
**Pull request by mreid-moz**
r?(@harterrt) for whether this seems like a reasonable place to put this type of documentation.
r?(@fbertsch) for completeness/correctness.

Thanks!

### CODE REVIEW

File: Projects.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+|-----------------------------------|-----------------------------------------|
+| [emr-bootstrap-spark][eb_spark]   | AWS bootstrap scripts for Spark.
+| [emr-bootstrap-presto][eb_presto] | AWS bootstrap scripts for Presto.
+| [emr-bootstrap-hbase][eb_hbase]   | AWS bootstrap scripts for HBase.
+
+[eb_spark]: https://github.com/mozilla/emr-bootstrap-spark
+[eb_presto]: https://github.com/mozilla/emr-bootstrap-presto
+[eb_hbase]: https://github.com/mozilla/emr-bootstrap-hbase
+
+## Data applications
```
Comment (fbertsch) [inline Projects.md]: Hardware report needs added, https://github.com/mozilla/firefox-hardware-report/

File: Projects.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+| Name and repo               | Description                       |
+|-----------------------------|-----------------------------------|
+| [python_moztelemetry][pymt] | Python APIs for Mozilla Telemetry
+| [moztelemetry][mt]          | Scala APIs for Mozilla Telemetry
+
+[pymt]: https://github.com/mozilla/python_moztelemetry
+[mt]: https://github.com/mozilla/moztelemetry
+
+## ETL code and Datasets
```
Comment (fbertsch) [inline Projects.md]: telemetry-airflow needs mentioned here or in `Infrastructure`.

File: Projects.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# Projects
+
+## Telemetry APIs
```
Comment (fbertsch) [inline Projects.md]: spark-hyperloglog belongs here, as well as presto-hyperloglog.

File: Projects.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+|------------------------------|-----------------------------------------|
+| [firefox-data-docs][docs_gh] | All the info you need to [answer questions about Firefox users with data][docs]
+| [reports.t.m.o][rtmo_gh]     | Knowledge repository for [public reports][rtmo]
+
+[docs_gh]: https://github.com/mozilla/firefox-data-docs
+[docs]: https://docs.telemetry.mozilla.org
+[rtmo_gh]: https://github.com/mozilla/mozilla-reports
+[rtmo]: https://reports.telemetry.mozilla.org
+
+### Non-public
```
Comment (fbertsch) [inline Projects.md]: Consider including the fx-data-interview repo here.

File: Projects.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+[taar]: https://github.com/mozilla/taar
+[ensemble]: https://github.com/mozilla/ensemble
+[shield]: https://wiki.mozilla.org/index.php?title=Firefox/Shield
+[tmo]: https://telemetry.mozilla.org
+[atmo]: https://analysis.telemetry.mozilla.org
+[stmo]: https://sql.telemetry.mozilla.org
+
+## Reference materials
+
+### Public
```
Comment (fbertsch) [inline Projects.md]: Should link to [firefox-source-docs telemetry section](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/). Perhaps the [Adding a new telemetry probe](https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Adding_a_new_Telemetry_probe), but I'm not as convinced about the latter.

File: Projects.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+[eb_spark]: https://github.com/mozilla/emr-bootstrap-spark
+[eb_presto]: https://github.com/mozilla/emr-bootstrap-presto
+[eb_hbase]: https://github.com/mozilla/emr-bootstrap-hbase
+
+## Data applications
+
+| Name and repo                   | Description                             |
+|---------------------------------|-----------------------------------------|
+| [telemetry.mozilla.org][tmo_gh] | Main entry point for viewing [aggregate Telemetry data][tmo]
```
Comment (fbertsch) [inline Projects.md]: [Cerberus](https://github.com/mozilla/cerberus) and [Medusa](https://github.com/mozilla/medusa) should be included here as well.

File: Projects.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+| [moztelemetry][mt]          | Scala APIs for Mozilla Telemetry
+
+[pymt]: https://github.com/mozilla/python_moztelemetry
+[mt]: https://github.com/mozilla/moztelemetry
+
+## ETL code and Datasets
+
+| Name and repo               | Description                           |
+|-----------------------------|---------------------------------------|
+| [telemetry-batch-view][tbv] | Scala ETL code for derived datasets
```
Comment (fbertsch) [inline Projects.md]: parquet2hive should be mentioned here or in `Infrastructure`.

File: Projects.md (hunk 2 above)
Comment (fbertsch) [inline Projects.md]: Also need [the aggregator](https://github.com/mozilla/python_mozaggregator)

Comment (fbertsch) [inline Projects.md]: Also need [telemetry-streaming](https://github.com/mozilla/telemetry-strreaming)

File: Projects.md (hunk 4 above)
Comment (mreid-moz) [reply Projects.md]: I don't think there's much point in linking to non-public resources here, so I just left the one `fx-data-planning` link, we can put other internal stuff in that repo.

**Comment by mreid-moz** [comment]
> Looking at all these repos is not, I think, very useful for onboarding. But they could be useful when writing a more complete overview of our infrastructure and processes.

This doc is meant as a reference for people to keep track of the large number of projects in flight (or find out more about a project name as it flies by). Does it seem useful in that sense?

File: Projects.md (hunk 3 above)
Comment (mreid-moz) [reply Projects.md]: I added spark-hyperloglog, which itself references presto-hll.

**Comment by mreid-moz** [comment]
@fbertsch RFAL, thanks for all those extra links!

**Comment by mreid-moz** [comment]
To be honest, I'd forgotten all about this repo until last week, so I'm happy to put this in a better home now.

Let's discuss and figure out the best place.

**Comment by mreid-moz** [comment]
Closing in favour of https://github.com/mozilla/firefox-data-docs/pull/69

**Pull request closed without merging**